bin/index.js

```diff
@@ -3,90 +3,60 @@
-        }, {
-            name: '🔒     Dealing with security',
-            value: ':lock:'
-        }, {
-            name: '⬆️     Upgrading dependencies',
-            value: ':arrow_up:'
-        }, {
-            name: '⬇️     Downgrading dependencies',
-            value: ':arrow_down:'
-        }, {
-            name: '👕     Removing linter warnings',
-            value: ':shirt:'
-        }, {
-            name: '🐧     Fixing something on Linux',
-            value: ':penguin:'
-        }, {
-            name: '🍎     Fixing something on Mac OS',
-            value: ':apple:'
-        }, {
-            name: '🏁     Fixing something on Windows',
-            value: ':checkered_flag:'
-        }, {
-            name: '🐥     Add or update config files like .gitignore',
-            value: ':hatched_chick:'
-        }, {
-            name: '⚡     Add a new feature',
-            value: ':zap:'
-        }, {
-            name: `🌵     I don't like emojis`,
-            value: ''
-        }]
-    }, {
-        type: 'input',
-        name: 'issue',
-        message: 'Issue number'
-    }
-];
-
-const writeCommitMessage = answers => {
-
-    let commitMsg = fs.readFileSync(process.argv[2]);
-
-    const emoji = answers.type;
-    const issue = answers.issue ? `#${answers.issue} ` : '';
-    commitMsg = `${emoji} ${issue}\n${commitMsg}`;
-
-    fs.writeFileSync(process.argv[2], commitMsg);
-
-    process.exit(0);
-};
-
-if(/COMMIT_EDITMSG/g.test(process.argv[2])) {
-    inquirer.prompt(questions, writeCommitMessage);
+const axios = require('axios');
+const chalk = require('chalk');
+const pathExists = require('path-exists');
+
+const gitmojis = 'https://raw.githubusercontent.com/carloscuesta/gitmoji/master/src/data/gitmojis.json';
+
+let questions = []
+
+if (process.argv[2] === '--init') {
+  if (pathExists.sync('.git')) {
+    const path = `${process.env.PWD}/.git/hooks`;
+
+    fs.writeFile(`${path}/prepare-commit-msg`, `#!/bin/sh\nexec < /dev/tty\ngitmoji-commit-hook $1`, (err) => {
+      if (err) {
+        console.error(chalk.red(`🚨  ERROR: ${err}`));
+      } else {
+        console.log(`${chalk.green('🎉  SUCCESS 🎉')}  gitmoji-commit-hook initialized with success.`);
+      }
+    });
+
+    fs.chmod(`${path}/prepare-commit-msg`, '755', (err) => {
+      if (err) {
+        console.error(chalk.red(`🚨  ERROR: ${err}`));
+      }
+    });
+  } else {
+    console.error(chalk.red('🚨  ERROR: The directory is not a git repository.'));
+  }
+} else {
+  axios.get(gitmojis)
+    .then(res => {
+      questions.push({
+        type: 'checkbox',
+        name: 'emoji',
+        message: 'Select emoji(s) for your commit',
+        choices: res.data.gitmojis.map(gitmoji => {
+          return {
+            name: gitmoji.emoji + '  ' + gitmoji.description,
+            value: gitmoji.emoji
+          };
+        })
+      });
+
+      if(/COMMIT_EDITMSG/g.test(process.argv[2])) {
+        inquirer.prompt(questions).then((answers) => {
```

Comment:
We should return `inquirer.prompt` so we can catch (line 59) any errors that inquirer might give.